apps/web/src/pages/JobPosterTemplates/JobPosterTemplatePage/components/KeyTasks.tsx

```diff
@@ -47,8 +47,8 @@ const KeyTasks = ({ jobPosterTemplateQuery }: KeyTasksProps) => {
         <div>
           {intl.formatMessage({
             defaultMessage:
-              "The following key tasks are examples of the work this role might perform. When drafting your job advertisement, you’re welcome to add, remove, or modify what is listed here to suit the specifics of your team and the job you’re hoping to fill.",
-            id: "KPenmQ",
+              "The following key tasks are examples of the work that might be part of this role. When drafting your job advertisement, you’re welcome to add, remove, or modify what is listed here to better fit your team and the job you’re hoping to fill.",
```

Comment:
While we are here. I'm not :100: certain but the single quotes in this paragraph do not look like single quotes and kind of stand out as different. Can we confirm and replace them if they are not? :sweat_smile:

Reply:
I don't see what you're referring to here..?

Reply:
It is very minor but the difference between ’ and ' 

While we can use either, we previously made the decision to always use plain single quotes for consistency.

Reply:
I'll be honest, I have no idea where these single quotes are

Reply:
> you’re welcome to add

and

> you’re hoping to fill

```suggestion
              "The following key tasks are examples of the work that might be part of this role. When drafting your job advertisement, you're welcome to add, remove, or modify what is listed here to better fit your team and the job you're hoping to fill.",
```

I would not commit this suggestion due to the id hash. As well, there are other places I noticed.

Reply:
Oh, you meant the apostrophe

Reply:
haha, yeah. sorry for the confusion

Reply:
Ouff, I didn't notice that before. Please go for consistency for the moment but I might advocate to move towards curly quotation marks/apostrophes at some point.

Reply:
Done in https://github.com/GCTC-NTGC/gc-digital-talent/pull/11609/commits/13d8de7facd0460722e0c184a6af1c204ad9233e